sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -306,8 +306,8 @@ def test_upretty_modifiers():
     assert upretty( Symbol('Fhat') ) == u('F̂')
     assert upretty( Symbol('Fbar') ) == u('F̅')
     assert upretty( Symbol('Fvec') ) == u('F⃗')
-    assert upretty( Symbol('Fprime') ) == u('F ̍')
-    assert upretty( Symbol('Fprm') ) == u('F ̍')
+    assert upretty( Symbol('Fprime') ) == u('F′')
+    assert upretty( Symbol('Fprm') ) == u('F′')
```

Comment:
So people can test how the characters render in their browser fonts:
Monospace: Prime `′` vs. single quote `'`
Proportional: Prime ′ vs. single quote '
For me, in the browser window for GitHub, the characters look the same in monospace but different in proportional. This can vary depending on installed fonts, and on program that displays them (browser, IDE, editor) and their respective font configuration; you may find that the difference is visible when editing but not in the browser, or vice versa.

I'm suggesting to change the prime symbol in the above string to `\N{PRIME}` so it's guaranteed to display differently for everybody. It's also testing the suffix detection in the pretty printer, not the visual display (which I suppose is the primary purpose of that test).
You may want to put that into a separate PR and do the same correction for Fhat, Fbar, Fvec etc.
To give a hint how it's supposed to look like, maybe something like this is in order:
`assert upretty( Symbol('Fbar') ) == u('F\N{COMBINING OVERLINE}') # F̅`
(For some reason, in a monospaced font, my browser moves the combining characters to the *following* character; seems to be a longstanding but evasive problem with Firefox.)